mx.go

```diff
@@ -12,8 +12,8 @@ type Mx struct {
 func (v *Verifier) CheckMX(domain string) (*Mx, error) {
 	domain = domainToASCII(domain)
 	mx, err := net.LookupMX(domain)
-	if err != nil {
-		return nil, ParseSMTPError(err)
+	if err != nil && len(mx) == 0 {
+		return nil, err
```

Comment:
It may be possible to keep the original handling of errors here, which does not affect the logic of the MX check